ansible/roles/kubernetes/tasks/debian.yml

```diff
@@ -0,0 +1,47 @@
+---
+# Instructions: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl
+- name: add apt signing key for kubernetes (1/6)
+  apt_key:
+    url: https://packages.cloud.google.com/apt/doc/apt-key.gpg
+    state: present
+
+# Removed the following repo due to stability issues: https://apt.kubernetes.io/
+- name: adding apt repository for kubernetes (2/6)
+  apt_repository:
+    repo: deb http://packages.cloud.google.com/apt/ kubernetes-xenial main
+    state: present
+  register: kubernetes_repository
+  retries: 10
+  until: kubernetes_repository is success
+
+# To downgrade these packages use the following apt options:
+#   dpkg_options: force-downgrade
+#   force: yes
+- name: install kubernetes packages (3/6)
+  apt:
+    name: '{{ packages }}'
+    state: present
+    update_cache: yes
+  register: apt_install_kube
+  retries: 5
+  until: apt_install_kube is success
+  vars:
+    packages:
```

Comment:
No need to use `vars.packages`, we can give the list of packages to be installed directly to `name`.

Reply:
Corrected.